js/components/tabs.js

```diff
@@ -75,33 +79,52 @@ var Tabs = React.createClass({
         var ref = ("tab-menu-" + (index + 1));
         var title = panel.props.title;
         var activeTabStyle = {};
+        var isSelected = this.state.tabActive === index + 1;
 
         var classes = ClassNames(
           'tabs-menu-item',
-          this.state.tabActive === (index + 1) && 'is-active',
-          'tabs-menu-item-' + index
+          'tabs-menu-item-' + index, {
+            'is-active': isSelected
+          }
         );
 
         //accent color
-        if (this.state.tabActive === (index+1) && this.props.skinConfig.general.accentColor) {
+        if (isSelected && this.props.skinConfig.general.accentColor) {
           var activeMenuColor =  "solid ";
           activeMenuColor += this.props.skinConfig.general.accentColor;
           activeTabStyle = {borderBottom: activeMenuColor};
         }
 
         return (
-          <li ref={ref} key={index} className={classes}>
-            <a onClick={this.setActive.bind(this, index + 1)} style={activeTabStyle} onMouseOver={this.highlight} onMouseOut={this.removeHighlight}>
+          <li ref={ref} key={index} className={classes} role={CONSTANTS.ARIA_ROLES.PRESENTATION}>
+            <AccessibleButton
+              style={activeTabStyle}
+              className="tabs-menu-item-btn"
+              ariaLabel={title}
+              ariaSelected={isSelected}
+              role={CONSTANTS.ARIA_ROLES.TAB}
+              onClick={this.setActive.bind(this, index + 1)}
+              onMouseOver={this.highlight}
+              onMouseOut={this.removeHighlight}
+              onFocus={this.onMenuItemFocus}>
               {title}
-            </a>
+            </AccessibleButton>
           </li>
         );
       }.bind(this));
 
     return (
-      <nav className='tabs-navigation' ref='tabsNavigation'>
-        <ul className='tabs-menu'>{menuItems}</ul>
-      </nav>
+      <div
+        className='tabs-navigation'
+        ref={function(e) { this.tabsNavigationElement = e }.bind(this)}
+        tabIndex="-1">
```

Comment:
Is this the same tabIndex used for the roving tab index (https://www.w3.org/TR/wai-aria-practices/#kbd_roving_tabindex)? This one has a capital I but the one used for the roving tab index has a lower case i. I see several instances of tabIndex with the capital I in the skin, so was curious about this.

Reply:
`tabIndex` with a capital I is the way it is declared within react. The actual html attribute is `tabindex`, so you'll see this whenever the code is working on the html element directly.

Reply:
Got it.